raiden/token_swap.py

```diff
@@ -666,9 +667,9 @@ def _run(self):  # pylint: disable=method-hidden,too-many-locals
 
             if log.isEnabledFor(logging.DEBUG):
```

Comment:
Does this `log.isEnabledFor` have to exist? I've seen a lot of it through the code, but shouldn't the logging suite take care of that on te log level configuration?

Reply:
The idea behind those is that it avoids all those `pex(...)` calls in case the log isn't going to be written. A possible alternative would be to use a custom class for addresses that has a `__str__` method but that's quite a bug change for probably little gain